cpu/native/periph/timer.c

```diff
@@ -27,6 +27,8 @@
 #include <mach/mach_init.h>
 #include <mach/mach_port.h>
 #include <mach/mach_host.h>
+/* Both OS X and RIOT typedef thread_t. timer.c does not use either thread_t. */
+#define thread_t riot_thread_t
```

Comment:
Now I'm confused... I thought one of the headers above includes OSX's `thread_t` and you are redefining it in this line.